lua/indent_blankline/init.lua

```diff
@@ -352,6 +368,15 @@ local refresh = function()
                     if offset + i == context_start then
                         context_indent = (indent or 0) + 1
                     end
+                    if
+                        last_normal_indent ~= -1 and
+                            indent + (extra and 1 or 0) > last_normal_indent + max_indent_increase
+                     then
+                        indent = last_normal_indent + max_indent_increase
+                        extra = false
```

Comment:
The `extra` check should only happen when `trail_indent` is true.
Also, `blankline` here works the same as `extra`. Either of them adds +1

Reply:
I am thinking about reverting to <https://github.com/lukas-reineke/indent-blankline.nvim/commit/de4b52818fbcadae9b3fac65cf909beed475ee0e> and renaming the variables.